minpy/utils/gradient_checker.py

```diff
@@ -43,7 +47,8 @@ def quick_grad_check(fun,
     grad_fun = grad(fun)
     unary_fun = lambda x: fun(arg + x * random_dir).asnumpy()
     numeric_grad = (unary_fun(eps / 2) - unary_fun(-eps / 2)) / eps
-    analytic_grad = np.sum(grad_fun(arg).asnumpy() * random_dir)
+    # FIXME: `grad_fun` will return a tuple, after fix, please remove the pylint disable
+    analytic_grad = np.sum(grad_fun(arg).asnumpy() * random_dir) # pylint: disable= no-member
```

Comment:
Sorry, I'm not very clear about this part. The `grad_fun` will return a tuple, is it right to change it to `grad_fun(arg)[0]`? @jermainewang @hotpxl